Makefile

```diff
@@ -0,0 +1,14 @@
+# Set these to the desired values
+ARTIFACT_ID=ldap
+VERSION=1.3.0
+# overwrite ADDITIONAL_LDFLAGS to disable static compilation
+# this should fix https://github.com/golang/go/issues/13470
+ADDITIONAL_LDFLAGS=""
```

Comment:
`ADDITIONAL_LDFLAGS` account for a project's go compilation. Since we do not do anything related, these lines are superfluous and should be left out.

Reply:
done